app/uk/gov/hmrc/agentsubscriptionfrontend/models/AmlsSubscriptionRecord.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscriptionfrontend.models
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.agentsubscriptionfrontend.models.FormBundleStatus.FormBundleStatus
+
+import java.time.LocalDate
+
+object FormBundleStatus extends Enumeration {
```

Comment:
To be honest I don't like using "enumerations" very much, I much prefer plain sealed traits even at the cost of a bit more boilerplate.
Are enumerations already in wide use in this project or in related projects?

Reply:
no not really but I don't see the problem with Enumerations...they are better than a String? which is all these Enum's represent really but happy to change it if you prefer